coda/coda_replication/urls.py

```diff
@@ -4,9 +4,9 @@
 urlpatterns = [
     url(r'^APP/queue/(?P<identifier>ark:\/\d+\/.+?)/$', views.queue, name='app-detail'),
     url(r'^APP/queue/$', views.queue, name='app-list'),
-    url(r'^queue/$', views.queue_recent, name='index'),
-    url(r'^queue/(?P<identifier>ark:\/\d+\/.+?)/$', views.queue_html, name='detail'),
-    url(r'^queue/search/$', views.queue_search, name='search'),
+    url(r'^queue/$', views.queue_recent, name='replication_index'),
```

Comment:
Since existing names use `-` not `_`, what do you think of changing all the `name` values you used `_` in to use `-` for consistency?

Reply:
sure! will do.